tests/templates/kuttl/smoke/32-assert.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: test-env-overrides
+commands:
+  - script: kubectl exec -n $NAMESPACE -c opa svc/test-opa-server -- env | grep SERVER_ROLE_GROUP_LEVEL_ENV_VAR
```

Comment:
Do you also want to assert that the role level env is set? :)
No need to change, I just though that asserting the env var on the Pod is a a bit more "kuttl-native" assert

Reply:
changelog: https://github.com/stackabletech/opa-operator/pull/754/commits/0ceaabb3ce3120dae7868d24209e70a7370f9fc9

check for role level: https://github.com/stackabletech/opa-operator/pull/754/commits/347c01f39a6c7713e4ef2236a8e9c417cbfb3466

I thought about that as well but i think you have to provide a complete list of env vars (which means adapting this test whenever you add another env var in code?)